src/java/us/kbase/scripts/test/TypeGeneratorTest.java

```diff
@@ -989,30 +988,33 @@ private static File prepareWorkDir(int testNum) throws IOException {
 		return workDir;
 	}
 
-	private static void runClientTest(int testNum, String testPackage, JavaData parsingData, 
-			File libDir, File binDir, int portNum, boolean needClientServer, File outDir,
-			String serverType) throws Exception {
-	    ensureCasperJsInstalled();
-	    System.out.println("- Java client -> " + serverType + " server");
-	    runJavaClientTest(testNum, testPackage, parsingData, libDir, binDir, portNum, needClientServer);
-	    if (outDir != null) {
-            String resourceName = "Test" + testNum + ".config.properties";
-            String clientConfigText = checkForClientConfig(resourceName);
-	        if (clientConfigText == null) {
-	            System.err.println("- Perl/Python/JavaScript client tests are skipped (" + resourceName + "not found)");	
-	            return;
-	        }
-	        if (clientConfigText.isEmpty())
-                return;
-	        System.out.println("- Perl client -> " + serverType + " server");
-	        runPerlClientTest(testNum, testPackage, parsingData, portNum, needClientServer, outDir);
-	        System.out.println("- Python client -> " + serverType + " server");
-            runPythonClientTest(testNum, testPackage, parsingData, portNum, needClientServer, outDir, false);
-            System.out.println("- Python3 client -> " + serverType + " server");
-            runPythonClientTest(testNum, testPackage, parsingData, portNum, needClientServer, outDir, true);
-            System.out.println("- JavaScript client -> " + serverType + " server");
-            runJsClientTest(testNum, testPackage, parsingData, portNum, needClientServer, outDir);
-	    }
+	private static void runClientTest(
+			final int testNum,
+			final String testPackage,
+			final JavaData parsingData, 
+			final File libDir,
+			final File binDir,
+			final int portNum,
+			final boolean needClientServer,
+			final File outDir,
+			final String serverType)
+			throws Exception {
+		System.out.println("- Java client -> " + serverType + " server");
+		runJavaClientTest(testNum, testPackage, parsingData, libDir, binDir, portNum, needClientServer);
+		if (outDir != null) {
+			String resourceName = "Test" + testNum + ".config.properties";
+			String clientConfigText = checkForClientConfig(resourceName);
+			if (clientConfigText == null) {
+				System.err.println("- Python client tests are skipped (" + resourceName + "not found)");
+				return;
+			}
+			if (clientConfigText.isEmpty())
+				return;
+			System.out.println("- Python client -> " + serverType + " server");
+			runPythonClientTest(testNum, testPackage, parsingData, portNum, needClientServer, outDir, false);
```

Comment:
So, out of scope here, but should we even be supporting Python2 vs Python3 at this point?
Will there be a culling, or just prop this up enough to make it run and move on? I'm ok with either.

Reply:
Once the tests pass this codebase shall be as corn before my sickle